client/my-sites/plugins/marketplace-footer/styles.module.css

```diff
@@ -0,0 +1,37 @@
+.marketplace-footer {
+	--color-accent: var( --studio-blue-50 );
+	--color-accent-60: var( --studio-blue-60 );
+	margin-bottom: -32px;
+
+    & .marketplace-footer__section::before {
+        background-color: #f6f7f7;
+    }
+
+    &:not(.is-logged-in) .marketplace-footer__section {
```

Comment:
Looks like the modifier could've been set directly on `marketplace-footer__section` and not on `marketplace-footer`.

Reply:
> Looks like the modifier could've been set directly on `marketplace-footer__section` and not on `marketplace-footer`.

Updated in fd65ebf19ef.